striprtf/striprtf.py

```diff
@@ -111,16 +111,15 @@ def rtf_to_text(text, encoding="utf-8", errors="strict"):
     text : str
         The rtf text
     encoding : str
-        Output encoding is fixed to "utf-8". It remains
-        in the signature to not break existing installations.
+        Output encoding, default is "utf-8"
```

Comment:
As I understand it, this isn't the *output* encoding, it's the *input* encoding — it's used to interpret hex codes in the RTF file.

(I'm also not sure why this should default to UTF-8 — aren't UTF-8 RTF files (codepage 65001) quite rare?)

Reply:
You are right, I will change the description to: 
```
    encoding : str
        Input encoding hint which is ignored if a explicit codepage is setin the file. 
        It is set to ansi encoding because it is most commonly used.
```
Also the default makes no sense, most of the rtf files will probably be ansi encodings.